app/src/main/java/com/example/caregiver/ArrayCheckboxAdapter.java

```diff
@@ -55,7 +55,7 @@ public View getView (int position, View convertView, ViewGroup parent)
 
         // Add the descriptive text
         TextView text = new TextView(this.getContext());
-        text.setText(this.getItem(position).toString());
+        text.setText("   "  + this.getItem(position).toString());

```

Comment:
Just adding space between the title of the task with the checkbox on Set Task page.